### PR TITLE
fix(app, components, protocol-desinger): handle color overrides for column 3

### PR DESCRIPTION
**Pull request by koji**
# Overview
[Design]
https://www.figma.com/file/bJxzgdG6IUVuN2eHZs8dBI/Design-Variables-Playground?type=design&node-id=1732-219864&mode=design&t=dIkX6zu7IsJRg7V6-0

Also I add new consts for `OVERFLOW-WRAP` to `components`.

There are a couple of things that need to be confirmed.
1. grey20's hex code
2. pd off deck slideout design
3. update app modal design

close RAUT-933

# Test Plan

# Changelog

# Review requests

# Risk assessment

### CODE REVIEW

**Comment by koji** [comment]
- [x] merge https://github.com/Opentrons/opentrons/pull/14354 for `grey20`
- [x] update `grey20` part

**Comment by mjhuff** [comment]
> * [ ]  merge [refactor(app): overrides dropdowns, cards, tiles, tabs colors #14354](https://github.com/Opentrons/opentrons/pull/14354) for `grey20`

Talked w/ Nick, this should be merged now.

File: app/src/pages/ProtocolSetup/index.tsx [hunk 1]
```diff
@@ -198,7 +199,7 @@ export function ProtocolSetupStep({
             name="more"
             size="3rem"
             // Required to prevent inconsistent component height.
-            style={{ backgroundColor: disabled ? 'transparent' : 'initial' }}
+            style={{ backgroundColor: 'initial' }}
```
Comment (koji) [inline app/src/pages/ProtocolSetup/index.tsx:202]: In this case, `disabled` is always `false`.

**Review by mjhuff — APPROVED** [review]
LGTM! Merging this in for 7.2 code completion.

**Pull request merged**